src/pages/gateway/release-notes.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Release notes
+description: This page lists changes that were made in each version of API Mesh for Adobe Developer App Builder.
+---
+
+# Release notes
+
+The following sections indicate when updates were made to API Mesh for Adobe Developer App Builder. Refer to the [Migration page](migration.md) for more information on upgrading versions.
+
+## Jan 17, 2023
+
+We have made the following changes since the API Mesh beta:
+
+### Enhancements
+
+- The new [`aio api-mesh:status` command](command-reference.md#aio-api-meshstatus) allows you to: <!-- the target of the link above is added in PR #75 -->
+  - View where your mesh is in the provisioning process
+  - See any provisioning errors associated with your mesh
+- Error messages will now contain `RequestIds`. You can provide these IDs to support when troubleshooting errors.
+- Responses now include response time information.
+
+### Bug Fixes
```

Comment:
let's add 2,3 known issues here
Do we have an FAQ by the way?

Reply:
We do not have an FAQ.